name.abuchen.portfolio/src/name/abuchen/portfolio/snapshot/ClientIndex.java

```diff
@@ -183,9 +193,13 @@ private void collectTransferalsAndTaxes(Interval interval)
                                         break;
                                     case BUY:
                                         addValue(buys, t.getCurrencyCode(), t.getAmount(), interval, d);
+                                        Money feeBuy = t.getUnitSum(Type.FEE);
+                                        addValue(fees, feeBuy.getCurrencyCode(), feeBuy.getAmount(), interval, d);
```

Comment:
Fees can also occur with deliveries (inbound and outbound).
I propose to handle the fees just like the tax in line 172 (see above):

```
// collect taxes
addValue(taxes, t.getCurrencyCode(), t.getUnitSum(Unit.Type.TAX).getAmount(), //
                                                interval, d);
```

Reply:
Oh, I missed this type of transaction. Thank you for the hint and fixing it.